doc/api/cli.markdown

```diff
@@ -0,0 +1,161 @@
+# Command Line Options
+
+<!--type=misc-->
+
+Node.js comes with a wide variety of CLI options. These options expose built-in debugging, multiple ways to execute scripts, and other helpful runtime options.
```

Comment:
@chrisdickinson Worded it this way so that it didn't sound like "multiple" applied to everything.